include/lbann/layers/transform/crop.hpp

```diff
@@ -140,15 +142,15 @@ class crop_layer : public transform_layer {
   protected:
 
   void fp_compute() override {
-    if(this->m_using_gpus) {
+    if (this->m_cudnn != nullptr) {
```

Comment:
I think that you should switch to a predicate like:

if(this->using_gpus()) {

to be more consistent with other part of the code.